Flow Designer: Revising the tutorial to help users avoid two problems (#127)
mulesoft/docs-design-center: modules/ROOT/pages/salesforce-to-twilio.adoc

== Save the opportunity data into a variable

Because you’ll be using data from the opportunity later in the flow, and cards that follow the trigger will be creating their own outputs, you need to store all of the opportunity data into a variable.

. Mouse over the arrow between the Salesforce card and the Transform card to change the arrow to a plus sign. Then, click the plus sign to add a new component.
. Search for and select the ‘Set Variable’ component
. Name the variable `OppDetails`.
. Click *Custom Expression* in the list of items on the right side of the card and drag it into the *Value* field.
. Click the arrow on the right side of the expression box and click *EXPAND*.
. In the  Custom Expression dialog, type `payload` in the *Expression* field.
+

> The expression `payload` indicates the entire input that arrives at the Set Variable card during one activation of the Mule app. As you can see in the right-hand pane of the card, the payload consists in this case of four data elements. It's possible to specify use one or more of those in the *Expression* box, rather than the expression `payload`. You could click and drag those elements into the box. However, for this tutorial, we're using the full payload.

Reviewer: ```suggestion
The expression payload indicates the entire input that arrives at the Set Variable card during one activation of the Mule app. The pane on the right side of the card shows, in this case, four data elements. It's possible to specify one or more of those in the *Expression* box, rather than the expression `payload`. You could click and drag those elements into the box. However, this tutorial uses the full payload.
```

+
.The four data elements that make up the payload that arrives at the Set Variable card.
image::fd-tutorial-set-var-right-pane.png[]

> . Click *OK* and then close the card. Your flow should look like this:

Reviewer: ```suggestion
. Click *OK* and then close the card. 
Your flow should look like this:
```
